### PR TITLE
Tests/styles us10

### PR DESCRIPTION
**Pull request by migueljcpinto**

### CODE REVIEW

File: components/NurseSelection/NurseSelection.js [hunk 1]
```diff
@@ -18,70 +18,79 @@ export default function NurseSelection({
...
-  if (selectedRole) {
-    filteredNurses = filteredNurses.filter(
-      (nurse) => nurse.role === selectedRole
-    );
-  }
-
-  if (showSpecialistsOnly) {
-    filteredNurses = filteredNurses.filter((nurse) => nurse.specialist);
-  }
+  const filteredNurses = useMemo(() => {
```
Comment (Kheirah) [inline components/NurseSelection/NurseSelection.js:21]: `useMemo` ❤️

**Review by Kheirah — CHANGES_REQUESTED** [review]
Looks good! Except that the deployment on Vercel fails for some reason. You'll need to resolve that issue before moving on to QA.

**Comment by donacrisante** [comment]
Hi Miguel,
the styling is great, but the update function doesn't work. When I want to save the changes, the window stays open and the entry is not updated. Please check this.

**Comment by donacrisante** [comment]
Looks good! Just correct the sentence under "Vacations" with: "Add Vacations, at least two days in a row. For just one day, choose a day off!"

**Pull request merged**